galley/pkg/crd/validation/config.go

```diff
@@ -390,14 +403,32 @@ func (whc *WebhookConfigController) reconcile(stopCh <-chan struct{}) {
 			// rebuild the desired configuration and reconcile with the
 			// existing configuration.
 			if err := whc.rebuildWebhookConfig(); err == nil {
-				whc.createOrUpdateWebhookConfig()
+				if retry := whc.createOrUpdateWebhookConfig(); retry {
+					configTimerC = time.After(retryUpdateAfterFailureTimeout)
+					if !retrying {
+						retrying = true
+						log.Info("webhook create/update failed - retrying")
```

Comment:
Should we print this every retry, or alternately include the retry interval here?

Reply:
`createOrUpdateWebhookConfig` already logs the per-call failure. Adding the retry interval here sounds good.